src/neuroconv/datainterfaces/ophys/inscopix/inscopixsegmentationdatainterface.py

```diff
@@ -0,0 +1,251 @@
+
+            desc_parts = [plane_desc]
+            if device_info.get("microscope_focus"):
+                desc_parts.append(f"Focus: {device_info['microscope_focus']} µm")
+            if device_info.get("exposure_time_ms"):
+                desc_parts.append(f"Exposure: {device_info['exposure_time_ms']} ms")
+            if device_info.get("microscope_gain"):
+                desc_parts.append(f"Gain: {device_info['microscope_gain']}")
+
+            imaging_plane_metadata["description"] = "; ".join(desc_parts)
+
+            # Sampling frequency
+            sampling_frequency = extractor.get_sampling_frequency()
+            if sampling_frequency:
+                imaging_plane_metadata["imaging_rate"] = sampling_frequency
+
+            # Optical channel
+            optical_channel = {
+                "name": "OpticalChannelDefault",
+                "description": "Inscopix optical channel",
+                "emission_lambda": float("nan"),
+            }
+            if device_info.get("channel"):
+                channel = device_info["channel"]
+                optical_channel["name"] = f"OpticalChannel{channel.capitalize()}"
+                optical_channel["description"] = f"Inscopix {channel} channel"
+
+            # LED power (segmentation uses different field name)
+            if device_info.get("led_power_1_mw_per_mm2"):
+                led_power = device_info["led_power_1_mw_per_mm2"]
+                optical_channel["description"] += f" (LED power: {led_power} mW/mm²)"
+
+            imaging_plane_metadata["optical_channel"] = [optical_channel]
+
+        # Image segmentation (specific to segmentation interface)
+        segmentation_desc = "Inscopix cell segmentation"
+        if analysis_info and analysis_info.get("cell_identification_method"):
+            segmentation_desc += f" using {analysis_info['cell_identification_method']}"
+        if analysis_info and analysis_info.get("trace_units"):
+            segmentation_desc += f" with traces in {analysis_info['trace_units']}"
+
+        # Add number of ROIs to description
+        num_rois = extractor.get_num_rois()
+        segmentation_desc += f" ({num_rois} ROIs identified)"
+
+        metadata["Ophys"]["ImageSegmentation"]["description"] = segmentation_desc
+
+        # Subject metadata
+        subject_metadata = {}
+        has_any_subject_data = False
+
+        # Subject ID
+        if subject_info and subject_info.get("animal_id"):
+            subject_metadata["subject_id"] = subject_info["animal_id"]
+            has_any_subject_data = True
+
+        species_value = None
+        strain_value = None
+
+        if subject_info and subject_info.get("species"):
+            species_raw = subject_info["species"]
+            # If it contains genotype info or doesn't match format, put it in strain instead
+            if " " in species_raw and species_raw[0].isupper() and species_raw.split()[1][0].islower():
```

Comment:
Same comment about adding a comment (!) here with examples of species raw so future developers can understand why you are parsing like this.

Reply:
I think that the parsing of this is too brittle but let's leave it like this and fix it if and we get more examples.